src/content/docs/es/core-concepts/astro-syntax.mdx

```diff
@@ -0,0 +1,191 @@
+
+{visible ? <p>Show me!</p> : <p>Else show me!</p>}
+```
+
+### Etiquetas dinámicas
+
+También puedes usar etiquetas dinámicas ajustando una variable en una etiqueta HTML o el import de un componente.
+
+```astro title="src/components/DynamicTags.astro" /Element|(?<!My)Component/
+---
+import MyComponent from "./MyComponent.astro";
+const Element = 'div'
+const Component = MyComponent;
+---
+<Element>Hello!</Element> <!-- renders as <div>Hello!</div> -->
+<Component /> <!-- renders as <MyComponent /> -->
+```
+
+Cuando se usan etiquetas dinámicas:
+
+- **Los nombres de las variables deben estar capitalizados** Por ejemplo, usa `Element`, y no `element`. Del contrario, Astro intentará renderizar el nombre de tu variable como una etiqueta HTML literal.
+
+- **Hidratación directivas no está soportada** Cuando se usa [`client:*` hydrataciones directivas](/es/core-concepts/framework-components/#hydrating-interactive-components), Astro necesita saber que componentes empaquetar para producción, y el patrón de la etiqueta dinámica previene esto de funcionar.
+
+### Fragmentos & Múltiples Elementos
+
+Un componente de Astro puede renderizar múltiples elementos sin la necesidad de envolver todo en un solo `<div>` o `<>`, no como JavaScript o JSX.
+
+```astro title="src/components/RootElements.astro"
+---
+// Template with multiple elements
+---
+<p>No need to wrap elements in a single containing element.</p>
+<p>Astro supports multiple root elements in a template.</p>
+```
+
+Sin embargo, cuando se usa una expresión para crear dinámicamente multiples elementos, deberías envolver estos elementos dentro de un **fragmento** como lo harías en JavaScript o JSX. Astro soporta ya sea `<Fragment> </Fragment>` o la taquigrafía `<> </>`.
+
+```astro title="src/components/FragmentWrapper.astro" "<>" "</>"
+---
+const items = ["Dog", "Cat", "Platypus"];
+---
+<ul>
+  {items.map((item) => (
+    <>
+      <li>Red {item}</li>
+      <li>Blue {item}</li>
+      <li>Green {item}</li>
+    </>
+  ))}
+</ul>
+```
+
+Los fragmentos también pueden ser utilizados para evitar elementos envolvedores cuandos e añade [directivas `set:*`](/es/reference/directives-reference/#sethtml), como se en el siguiente ejemplo:
+
+```astro title="src/components/SetHtml.astro" "Fragment"
+---
+const htmlString = '<p>Raw HTML content</p>';
+---
+<Fragment set:html={htmlString} />
+```
+
+### Astro diferencia entre Astro y JSX
+
+La sintaxis de los componentes Astro son un sobrepuesto de HTML. Fue diseñado para sentirse familiar con cualquiera con experiencia en HTML o JSX, pero hay unas cuantas diferencias entre los archivos `.astro` y JSX.
+
+#### Atributos
+
+En Astro, puedes usar el formato estandarte `kebab-case` para todos los atributos HTML en vez de `camelCase` usado en JSX. Esto también funciona para `class`, que no es soportado por React.
+
+```jsx del={1} ins={2} title="example.astro"
+<div className="box" dataValue="3" />
+<div class="box" data-value="3" />
+```
+
```

Comment:
tendrias que añadir una nueva sección
```suggestion
#### Multiple Elements

An Astro component template can render multiple elements with no need to wrap everything in a single `<div>` or `<>`, unlike JavaScript or JSX.

```astro title="src/components/RootElements.astro"
---
// Template with multiple elements
---
<p>No need to wrap elements in a single containing element.</p>
<p>Astro supports multiple root elements in a template.</p>
```
```